oshi-core/src/main/java/oshi/driver/linux/proc/ProcessStat.java

```diff
@@ -40,8 +43,152 @@
 @ThreadSafe
 public final class ProcessStat {
 
+    /**
+     * Constant defining the number of integer values in {@code /proc/pid/stat}. 2.6
+     * Kernel has 44 elements, 3.3 has 47, and 3.5 has 52.
+     */
+    public static final int PROC_PID_STAT_LENGTH;
     private static final Pattern DIGITS = Pattern.compile("\\d+");
 
+    static {
```

Comment:
There's a lot of stuff moved here, where there any other changes?  Any reason to rearrange the order?   Generally like to keep big whitespace changes out of diffs to ease review.

Reply:
I guess it was the IDE which formatted it  :( . I applied the setting but seems it is not working.